### PR TITLE
Create corona-dao

### PR DESCRIPTION
**Pull request by orenyodfat**

### CODE REVIEW

File: daos-params/corona-dao [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+       "permissions": "0x00000000",
+       "alias": "ReputationAdmin",
+       "fromArc": true,
+       "arcVersion": "0.0.1-rc.40"
+     }
+   ],
+   "useDaoCreator": true,
+   "founders": [
+     {
+       "address": "0x74b4b39dAb95975D9A9DDB01F2e3335f873076AC",
```
Comment (orenyodfat) [inline daos-params/corona-dao]: @Theylon is that your address ?

File: daos-params/corona-dao [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+     {
+       "name": "ReputationAdmin",
+       "schemeName": "ReputationAdmin",
+       "isUniversal": false,
+       "params": [0, 1614764214000, 0],
+       "runFunctions": [
+         {
+           "functionName": "transferOwnership",
+           "params": [
+             "0x06044b5359D8dF7886366c22C61c7ecd29BECac7"
```
Comment (orenyodfat) [inline daos-params/corona-dao]: @Theylon is that you ?

File: daos-params/corona-dao (hunk 1 above)
Comment (Theylon) [reply daos-params/corona-dao]: Yes, it's the dHack address

File: daos-params/corona-dao (hunk 2 above)
Comment (Theylon) [reply daos-params/corona-dao]: yes yes, it's dHack wallet addresss

File: daos-params/corona-dao (hunk 1 above)
Comment (Theylon) [reply daos-params/corona-dao]: This should be changed to: 0x06044b5359D8dF7886366c22C61c7ecd29BECac7

Comment (orenyodfat) [reply daos-params/corona-dao]: done

**Pull request merged**